lib/chef/knife/supermarket_list.rb

```diff
@@ -37,25 +37,40 @@ class SupermarketList < Knife
         default: "https://supermarket.chef.io",
         proc: Proc.new { |supermarket| Chef::Config[:knife][:supermarket_site] = supermarket }
 
+      option :sort_by,
+        long: "--sort-by SORT",
+        description: "Use to sort the records",
+        in: %w{recently_updated recently_added most_downloaded most_followed}
```

Comment:
✨ Yes! Great guarding of [the actual options available in the API endpoint](https://github.com/chef/supermarket/blob/cc49068bc4d2affff11b77939b61e0d2bd6f9a29/src/supermarket/app/models/cookbook.rb#L30-L33). ✅